src/shared/components/common/loading-ellipses.tsx

```diff
@@ -15,7 +15,11 @@ export class LoadingEllipses extends Component<any, LoadingEllipsesState> {
   }
 
   render() {
-    return this.state.ellipses;
+    return (
+      this.state.ellipses || (
+        <span dangerouslySetInnerHTML={{ __html: "&nbsp;" }} />
+      )
+    );
```

Comment:
Why the space is ellipses doesn't exist? And why use `dangerouslySetInnerHTML` for this?

Reply:
Modlog title collapses when it cycles to the empty string

Looks like `<span>&nbsp;</span>` does work. Thought i tried that. I'm moving it to modlog directly.